package.json

```diff
@@ -54,7 +54,6 @@
     "rollup-plugin-istanbul": "^2.0.1",
     "rollup-plugin-node-resolve": "^3.4.0",
     "rollup-plugin-terser": "^3.0.0",
-    "watchify": "^3.9.0",
```

Comment:
I remember trying to remove that dependency, thinking it wasn't used anymore, but at that time it was a peer dependency (unfortunately, I can't remember for what package). Did you try to run all commands with `--watch` (when supported and after `npm prune`)?

Reply:
Yes, `build`, `docs`, and `test` all still work with `--watch`

Reply:
It was [`karma-browserify`](/nikku/karma-browserify/blob/master/package.json#L61), removed in #5904.